package.json

```diff
@@ -173,13 +173,14 @@
     "analyze-bundles": "webpack-bundle-analyzer stats.json public -h 0.0.0.0 -p 9898",
     "analyze-css": "node bin/analyze-css.js",
     "autoprefixer": "postcss -r --use autoprefixer",
+    "postcss": "postcss -r -u postcss-custom-properties -u autoprefixer -c postcss.config.json",
```

Comment:
It feels a bit weird to leave `autoprefixer` split out, but have `custom-properties` step to be part of a wrapping task.  Should we do one of:
* also split out `custom-properties`
* bring `autoprefixer` into the `postcss` task

...? Is there any reason we'd want to run either of those two things independently?

Reply:
Good point! The reason for the current state is that we'd only want to use an additional postcss step for the parts of the build that actually need it. Currently, we only make use of css custom properties in the main styles and not the sections. Thus the idea is to use it only for the build step of the main styles and don't increase the build time where it's not necessary.